domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3281,6 +3281,45 @@ void getSQLConnection(int queryTimeout, Handler<AsyncResult<SQLConnection>> hand
     });
   }
 
+  /**
+   * Execute the given function within a transaction.
+   * <p
+   * Similar to {@link PgPool#withTransaction(Function)}
```

Comment:
This link is fine, however, the key features of this method should be described here:
* The connection is automatically closed in all cases when the function exits.
* The transaction is automatically committed if the function returns a succeeded Future. The transaction is automatically roll-backed if the function returns a failed Future or throws a Throwable.
* The method returns a succeeded Future if the commit is successful, otherwise a failed Future.

Reply:
okie dokie. Copied verbatim to ul, li lines.